osu.Game/Screens/Play/Player.cs

```diff
@@ -158,7 +158,8 @@ protected override void OnEntering(GameMode last)
 
         protected override bool OnExiting(GameMode next)
         {
-            dimLevel.ValueChanged -= dimChanged;
+            if(dimLevel != null)

```

Comment:
it happens when the onexit in load is called, for example if you click play on a non standard beatmap